graylog2-web-interface/src/components/content-packs/ContentPackPreview.jsx

```diff
@@ -18,7 +18,7 @@ class ContentPackPreview extends React.Component {
   };
 
   _renderDownload() {
-    const contentPackEncoded = encodeURIComponent(JSON.stringify(this.props.contentPack, null, 2));
+    const contentPackEncoded = encodeURIComponent(JSON.stringify(this.props.contentPack.toJSON(), null, 2));
```

Comment:
You do not need to call `toJSON` explicitly, `JSON.stringify` is doing it [implicitly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Description).

Reply:
I did not know that! Thank you!